.github/workflows/pr_build.yaml

```diff
@@ -485,3 +485,14 @@ jobs:
         with:
           name: binaries
           path: ./artifacts/
+
+  success:
+    runs-on: ubuntu-20.04
```

Comment:
I noticed the other commands have a `name` for displaying on the GitHub UI. Do we need one here as well?

Reply:
no, the name is implied from the job if not set explicitly